data_steward/validation/metrics/required_labs.py

```diff
@@ -54,6 +80,67 @@ def load_measurement_concept_sets_descendants_table(project_id, dataset_id):
     :return: None
     """
 
+    descendants_table_name = f'{project_id}.{dataset_id}.{MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE}'
+
+    client = bq.get_client(project_id)
+    dataset = client.dataset(dataset_id)
+    vocab_dataset = client.dataset(common.VOCABULARY_DATASET)
+
+    # concept table and concept ancestor table source tables
+    concept_source_table = vocab_dataset.table(common.CONCEPT)
+    concept_ancestor_source_table = vocab_dataset.table(common.CONCEPT_ANCESTOR)
+
+    # concept table and concept ancestor table destination tables
+    concept_dest_table = dataset.table(common.CONCEPT)
+    concept_ancestor_dest_table = dataset.table(common.CONCEPT_ANCESTOR)
+
+    descendants_table_ref = dataset.table(
+        MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE)
+    concept_table_ref = dataset.table(common.CONCEPT)
+    concept_ancestor_table_ref = dataset.table(common.CONCEPT_ANCESTOR)
+
+    # query will check the row counts of each table in the specified dataset
+    row_count_query = ROW_COUNT_QUERY.render(project_id=project_id,
+                                             dataset_id=dataset_id)
+
+    job = client.query(row_count_query)
```

Comment:
This part can be in a new function, like `check_and_copy_tables` or so. This will help make it unit-testable.